chips/README.md

```diff
@@ -0,0 +1,200 @@
+# Chip components
+
+Chips appear dynamically as a group of multiple interactive elements. With chips, users can filter
+content, enter information, make selections or trigger actions.
+
+## Chip design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:chip:<versions>")
+```
+
+### Action
+
+```kotlin
+object VitaminChips {
+
+    @ExperimentalMaterialApi
+    @Composable
+    fun Action(
+        label: String,
+        onClick: () -> Unit,
```

Comment:
According to our conventions page, callbacks should be in the last parameters: https://github.com/Decathlon/vitamin-compose/blob/main/CONVENTIONS.md#parameter-ordering

Reply:
Apply these conventions to all components! :)